### DISCUSSION TITLE
Is download working?

### DISCUSSION THREAD

**Discussion by MatthiasRInst** [wixtoolset/issues#8213] [Questions]
Hello,

These days I am not able to download the packages I need.
The "Download" graphical link on wixtoolset.org leads me to the docs area, not to a download page.
Here on github, [https://github.com/orgs/wixtoolset/packages](https://github.com/orgs/wixtoolset/packages) shows me the available packages. When I go to the page for one package (e.g. WixToolset.Util.wixext), I am shown the available versions, but cannot find a download link either.

Am I missing something?

Best regards

Matthias Reuss

**Comment by wmanning** [comment]
Wix4 and 5 are now on Nuget.  That's how I get them.  Other methods, I'm not sure.

**Comment by MatthiasRInst** [comment]
Thank you, I could get them from nuget.org meanwhile.

But:
Download links on wixtoolset.org are missing (or broken?), as described above.

**Reply by robmen** [reply]
Which links on wixtoolset.org are missing or broken?

**Reply by MatthiasRInst** [reply]
wixtoolset.org, start page, center.
There is the big "download" link with the down arrow (and the "latest releases" button below).

This link does not link to a download page (e.g. nuget.org), but to docs/intro, where there is no´obvious download link.

**Reply by robmen** [reply]
So the confusion is the word "Download"? You found it set up an expectation for a file to download (like with a web browser) vs using the command like `dotnet tool` to download the WiX Toolset?

**Reply by MatthiasRInst** [reply]
Yes I did. The docs/intro page does mention `dotnet tool` as the command to *install* the tool, but I do not see that as an obvious *download* command.
